phy/datasets/mock.py

```diff
@@ -21,10 +25,91 @@ def artificial_waveforms(n_spikes=None, n_samples=None, n_channels=None):
     return .25 * nr.normal(size=(n_spikes, n_samples, n_channels))
 
 
+def artificial_features(n_spikes=None, n_features=None):
+    return .25 * nr.normal(size=(n_spikes, n_features))
+
+
+def artificial_masks(n_spikes=None, n_channels=None):
+    return nr.uniform(size=(n_spikes, n_channels))
+
+
 def artificial_traces(n_samples, n_channels):
     # TODO: more realistic traces.
     return .25 * nr.normal(size=(n_samples, n_channels))
 
 
 def artificial_spike_clusters(n_spikes, n_clusters, low=0):
-    return nr.randint(size=n_spikes, low=low, high=n_clusters)
+    return nr.randint(size=n_spikes, low=low, high=max(1, n_clusters))
+
+
+def artificial_spike_times(n_spikes):
+    # TODO: switch from sample to seconds in the way spike times are
+    # represented throughout the package.
+    return np.cumsum(nr.randint(low=0, high=100, size=n_spikes))
+
+
+#------------------------------------------------------------------------------
+# Artificial Experiment
+#------------------------------------------------------------------------------
+
+class MockExperiment(BaseExperiment):
+    n_channels = 32
+    n_features = 32 * 3
```

Comment:
can we try with a different n_features and n_channels to the regular ones? it's most likely to go wrong in cases with few users...